Sources/Build/BuildDelegate.swift

```diff
@@ -269,9 +294,9 @@ public final class BuildDelegate: BuildSystemDelegate, SwiftCompilerOutputParser
         self.outputStream = outputStream as? ThreadSafeOutputByteStream ?? ThreadSafeOutputByteStream(outputStream)
         self.progressAnimation = progressAnimation
         self.buildExecutionContext = bctx
-        self.swiftParsers = bctx.buildDescription.swiftTargetMap.mapValues {
+        self.swiftParsers = bctx.buildDescription?.swiftTargetMap.mapValues {
```

Comment:
Is this correct, or is this something not implemented? Its not at all obvious how this can be as "optional" as it is, deserves a comment regardless

Reply:
Will add comment. This is optional because we won't have a build description when building the "PackageStructure" target. Well, we would have one but it might be stale.